doc/api/events.md

```diff
@@ -1846,8 +1846,11 @@ Sets the `defaultPrevented` property to `true` if `cancelable` is `true`.
 added: v14.5.0
 -->
 
+> Stability: 3 - Legacy: Use [`event.defaultPrevented`][] instead.
+
 * Type: {boolean} True if the event has not been canceled.
 
+Returns a value which is opposite of the value returned by `defaultPrevented`.
```

Comment:
```suggestion
The value of `event.returnValue` is always the opposite of `event.defaultPrevented`.
```

Reply:
Applied your suggestion. Thank you for suggestion.